### PR TITLE
util String to Date

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/main/java/core/utils/StringToDate.java [hunk 1]
```diff
@@ -1,17 +1,92 @@
...
+                calendar.add(Calendar.DAY_OF_MONTH, +1);
+                return calendar.getTime();
+            }
+            String[] actions = cutString(valor);
+            if (actions[actions.length - 1].contains("ago")) {
+                return calculateDate(Integer.parseInt(actions[0]) * -1, actions[1], calendar);
+            } else {
+                return calculateDate(Integer.parseInt(actions[0]), actions[1], calendar);
+            }
+
```
Comment (cristianfchq) [inline src/main/java/core/utils/StringToDate.java]: this line should be removed

File: src/test/java/core/utils/StringToDateTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package core.utils;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
```
Comment (cristianfchq) [inline src/test/java/core/utils/StringToDateTest.java]: this line should be removed

File: src/main/java/core/utils/StringToDate.java [hunk 3]
```diff
@@ -40,30 +40,30 @@ public String convertirString(final String value) {
...
             calendar.add(Calendar.HOUR, value);
-            return dateFormat.format(calendar.getTime());
+            return calendar.getTime();
         }
         if (property.contains("min")) {
             calendar.add(Calendar.MINUTE, value);
-            return dateFormat.format(calendar.getTime());
+            return calendar.getTime();
         }
         if (property.contains("seg")) {
```
Comment (SaulCM) [inline src/main/java/core/utils/StringToDate.java]: Probably meant "sec" for seconds.

File: src/main/java/core/utils/StringToDate.java [hunk 4]
```diff
@@ -7,31 +7,31 @@
...
         try {
-
+            if (valor.contains("/")) return dateFormat.parse(valor);
             if (valor.equals("today")) {
-                return dateFormat.format(actualDate);
+                return actualDate;
             } else if (valor.equals("yesterday")) {
-                calendar.set(Calendar.DAY_OF_MONTH, calendar.get(Calendar.DATE) - 1);
-                return dateFormat.format(calendar.getTime());
+                calendar.add(Calendar.DAY_OF_MONTH, -1);
```
Comment (SaulCM) [inline src/main/java/core/utils/StringToDate.java:29]: Care with magic numbers.

**Review by SaulCM — APPROVED** [review]
Good job.

**Pull request merged**